### PR TITLE
python python-modules module - closes #668

### LINKED ISSUE #668: python python-modules module

### PR DESCRIPTION
**Pull request by BenHesketh21**

### CODE REVIEW

File: topics/python/modules/python-modules/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import boss
+
+bad_boss("Bill Steves")
+```
+
+This should return "Bill Steves is a bad boss!"
+
+## Exercises
+
+Create 2 new python files. In one file, write functions that generate 2 numbers between 1 and 6. Make this a module called `dice`.
```
Comment (LukeBenson) [inline topics/python/modules/python-modules/README.md]: This doesn't make sense to me. Do you want two functions that generate a number each?

File: topics/python/modules/python-modules/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+```python
+import math
+
+number = float(input("Enter a number:"))
+answer = math.sqrt(number)
+
+print(answer)
+```
+
```
Comment (BenHesketh21) [inline topics/python/modules/python-modules/README.md]:
```suggestion
This piece of code is importing the `math` module, then using the `sqrt` function to find the square root of the number the user enters. Then it prints the answer.
```

File: topics/python/modules/python-modules/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+def not_a_boss(name):
+    print(name + " is not even a boss")
+```
+
+```python
+# In app.py
+import boss
+
+bad_boss("Bill Steves")
```
Comment (BenHesketh21) [inline topics/python/modules/python-modules/README.md]:
```suggestion
boss.bad_boss("Bill Steves")
```

File: topics/python/modules/python-modules/README.md (hunk 1 above)
Comment (BenHesketh21) [inline topics/python/modules/python-modules/README.md]:
```suggestion
Create 2 new python files. In one file, write a function that will generate a number between 1 and 6. Make this a module called `dice`.
```

File: topics/python/modules/python-modules/README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+This should return "Bill Steves is a bad boss!"
+
+## Exercises
+
+Create 2 new python files. In one file, write functions that generate 2 numbers between 1 and 6. Make this a module called `dice`.
+
+In the second file, use the `dice` module to 'throw' the dice.
```
Comment (BenHesketh21) [inline topics/python/modules/python-modules/README.md]:
```suggestion
In the second file, use the `dice` module to simulate throwing 2 dice and return the values you get from the dice.
```

**Pull request merged**